src/frontend/src/components/public/PIAFormTabs/collectionUseAndDisclosure/index.tsx

```diff
@@ -13,10 +13,16 @@ import { faUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
 import { ColumnMetaData, Table } from '../../../common/Table';
 import { setNestedReactState } from '../../../../utils/object-modification.util';
 
-const PIACollectionUseAndDisclosure = () => {
+interface IComponentProps {
+  isReadOnly?: boolean;
+}
+
+const PIACollectionUseAndDisclosure = (props: IComponentProps) => {
   const [pia, piaStateChangeHandler] =
     useOutletContext<[IPiaForm, PiaStateChangeHandlerType]>();
```

Comment:
In the recent change, I worked with Bing to pass isReadOnly as a third parameter to the outlet. So, rather than you have to send from the router, you will get that value from the parent.

<img width="463" alt="image" src="https://user-images.githubusercontent.com/87394256/225356667-3924cf2f-275d-4f85-82cc-b4b0715a737a.png">